include/srtp.h

```diff
@@ -1046,6 +1046,95 @@ void srtp_crypto_policy_set_aes_cm_256_hmac_sha1_32(srtp_crypto_policy_t *p);
  */
 void srtp_crypto_policy_set_aes_cm_256_null_auth(srtp_crypto_policy_t *p);
```

Comment:
I'm all for keeping the API consistent, but I will note that here we have aes_cm_xxx and the variable names inside are aes_icm_xxx or AES_XXX_ICM.  Would be nice if those were all consistent.